checkUrlExists/lambda/package.json

```diff
@@ -7,10 +7,10 @@
   "author": "Dockstore",
   "license": "MIT",
   "dependencies": {
-    "node-libcurl": "^2.3.4"
+    "basic-ftp": "^5.0.3",
```

Comment:
FWIW, basic-ftp has more dependents and more downloads, so this is probably a good move in terms of maintainability 
https://www.npmjs.com/package/basic-ftp?activeTab=readme
https://www.npmjs.com/package/node-libcurl